src/SFA.DAS.LevyTransferMatching.Api/Controllers/PledgeController.cs

```diff
@@ -162,5 +167,25 @@ public async Task<IActionResult> JobRole()
                 return new StatusCodeResult((int)HttpStatusCode.InternalServerError);
             }
         }
+
+        [Authorize(Policy = PolicyNames.PledgeAccount)]
+        [HttpGet]
+        [Route("accounts/{accountId}/pledges/{pledgeId}/applications")]
+        public async Task<IActionResult> PledgeApplications(int pledgeId)
+        {
+            var applications = await _mediator.Send(new GetApplicationsQuery { PledgeId = pledgeId });
```

Comment:
The pattern we've used elsewhere has avoided multiple calls to the mediator from the controller in favour of a single call which does everything required and returns a single result. This approach also exposes an inner api type (GetStandardsListItem) nested inside of an outer api response type.

Reply:
So you would create a second object to hold the Standard data and convert from GetStandardsListItem to it prior to returning back from the query handler?